### ISSUE TITLE
can't build game

### ISSUE THREAD

**Issue by whizzkid** [pixijs/open-games#6]
I can't seem to **build** the bubbo-bubbo project.  

`npm run start`  works perfectly, no errors and I can play the game.

but when I try:

`npm run build` I get a load of errors.  I've listed the first 2 here. (it found 97 errors in 26 files.)

```code
src/assets.ts:5:5 - error TS2305: Module '"pixi.js"' has no exported member 'ResolverAssetsArray'.

5     ResolverAssetsArray,
      ~~~~~~~~~~~~~~~~~~~

src/game/entities/Bubble.ts:73:28 - error TS2345: Argument of type 'Container<DisplayObject>' is not assignable to parameter of type 'DisplayObject'.

73         this.view.addChild(this.bubbleView.view);
                              ~~~~~~~~~~~~~~~~~~~~
```

That doesn't seem right to me? :)

**Comment by CyberDex** [comment]
Will be fixed in #7

**Comment by bigtimebuddy** [comment]
This has been addressed. Closing.
